docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,311 @@ The ACS will be used for V&V testing. It is defined as:
+can be a bit complicated, so here are some examples: 
+
+- A single adult will have 1 row, regardless of the number of jobs they had 
+- Joint filers (a married couple) will have 1 row, regardless of the number of jobs 
+- A married couple with unemployed children will have 1 row for the whole family 
+- A married couple with an employed child might have 2 rows: one for the parents and child as a dependent, and a second for the child as an individual filing their own taxes 
+
+Here is a photo showing how this might look. Note that the three tables 
+are just 2 really long rows for two simulants. 
+
+.. image:: 1044_example.PNG
+
+.. todo::
+
+    Define a maximum number of dependents that a simulant can have 
+
+
+**Who to Sample** 
+
+.. todo::
+
+    Need to allow dependents (and possibly joint filing) with people outside the household. Key cases are college students, divorced parents, etc. 
+
+    Also need to address complex family structures 
+
+
+Not everyone who receives a W2 or 1099 will end up filing taxes. 
+However, those who do not are concentrated in low incomes for whom 
+taxes are not required. Currently, we will chose to have all those 
+who are legally required to file taxes, file taxes. This is a 
+limitation and is listed below. 
+
+For simulants that receive below the minimum income, 42.14% will 
+still file taxes. [Cilke_1998]_ The remainder will not. The minimum 
+income is based on the household structure and is listed in the table below. 
+We will not model persistence year to year. 
+
+**In the current model, no one will be low income, this will be changed later.** 
+
+.. list-table:: Minimum Income  
+  :widths: 20 20 
+  :header-rows: 0
+
+  * - Simulant Type 
+    - Minimum Income 
+  * - Single filing, under 65 
+    - $12,550 
+  * - Single filing, over 65 
+    - $14,250
+  * - Married joint filing, both under 65 
+    - $25,100
+  * - Married joint filing, one under 65 
+    - $26,450
+  * - Married joint filing, both over 65 
+    - $27,800
+  * - Married separate filing 
+    - $5
+
+
+Based on the household structure, the following rules can be applied 
+for who files taxes: 
+
+- Assume that 95% of spouses file jointly, this can be randomly assigned. [Nolo]_ Others will file separately. 
+    * The only spouses we will recognize are [Reference person, Opp-sex spouse] and [Reference person, Same-sex spouse]. 
+    * The reference person will submit the form, the spouse will be listed as the joint filer. 
+    * There does not need to be persistence in who files jointly, it can be re-drawn each year. 
+- All other non-married simulants in a household with a W2 or 1099 will file separately, based on the income rules above (e.g., a low-income earner in a house with other earners will be randomly assigned to file or not file, independent of others in the household). Please note that simulants can BOTH be claimed as a dependent AND file their own taxes. 
```

Comment:
just to make sure i'm following: 

- if person A can claim person B as a dependent, they always will.
- if person B had an employer + income in the past year, the only reason they won't submit a w2 or 1099 and a 1044 is if they have a sufficiently low income, and they are selected as among the 67% who still file taxes; is that correct? i.e., there aren't other reasons someone who is claimed as a dependent might not file taxes?

Reply:
Yes - this is correct! Anyone who can be claimed as a dependent will be claimed. Regardless of if you are claimed, you still decide to do or not do taxes independently. 

In real life, there is a whole "informal economy" where people have jobs that pay in cash and don't show in taxes. But we are not modeling this at all.